packages/block-editor/src/components/text/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import RichText from '../rich-text';
+
+function Text( props ) {
```

Comment:
Should we name this PlainText instead.

Reply:
See https://github.com/WordPress/gutenberg/pull/18361#issuecomment-552884790.